README.md

```diff
@@ -3,30 +3,46 @@
 # Here is the code for RCareWorld V1.0.0 🦾
 - Check the website https://emprise.cs.cornell.edu/rcareworld/
 
-We support Windows, Linux (Tested on Ubuntu), and MacOS. You need to make sure git and git-lfs is installed on your system. You should use powershell for windows, 
-and bash for Linux and MacOS.
+We support Windows, Linux (Tested on Ubuntu), and MacOS. You need to make sure git and git-lfs are installed on your system. You should use Git Bash if you have it (else Powershell) for windows, 
```

Comment:
Link to tutorial?

Reply:
Fixed by adding information on how to install Git and Git Large File Storage